platform/mellanox/mlnx-fw-upgrade.j2

```diff
@@ -19,9 +19,11 @@ declare -r EXIT_SUCCESS="0"
 declare -r EXIT_FAILURE="1"
 
 declare -r QUERY_CMD="mlxfwmanager --query"
+declare -r LIST_CONTENT_CMD="mlxfwmanager -l"
 declare -r BURN_CMD="mlxfwmanager -u -f -y"
 
 declare -r QUERY_FILE="/tmp/mlxfwmanager-query.log"
+declare -r LIST_CONTENT_FILE="/tmp/list-content.log"
```

Comment:
@ayurkiv-nvda please have `mlxfwmanager-list-content.log` to avoid I/O race conditions

Reply:
Done